package/ci/appveyor-desktop-mingw.bat

```diff
@@ -67,6 +67,7 @@ cmake .. ^
     -DCMAKE_INSTALL_PREFIX=%APPVEYOR_BUILD_FOLDER%/deps ^
     -DCMAKE_PREFIX_PATH=%APPVEYOR_BUILD_FOLDER%/openal;%APPVEYOR_BUILD_FOLDER%/devil ^
     -DWITH_ASSIMPIMPORTER=OFF ^
+    -DWITH_BASISIMPORTER=OFF -DBASISU_DIR=%APPVEYOR_BUILD_FOLDER%/basis_universal ^
```

Comment:
Would be good to mention the `OFF` above in the comment (due to mingw not compiling), otherwise we forget why is it disabled.